ci/build.py

```diff
@@ -116,9 +119,19 @@ def buildir() -> str:
     return os.path.join(get_mxnet_root(), "build")
 
 
+def default_ccache_dir() -> str:
+    if 'CCACHE_DIR' in os.environ:
+        ccache_dir = os.path.realpath(os.environ['CCACHE_DIR'])
+        os.makedirs(ccache_dir, exist_ok=True)
+        return ccache_dirpython
+    # Share ccache across containers (should we have a separate dir per platform?)
```

Comment:
No, ccache should be able to figure this out on its own, right?

Reply:
Maybe